### PR TITLE
Refresh generated code

### PR DESCRIPTION
**Pull request by AldanTanneo**
Not really necessary, just a bit of a refresh for the generated code, reflecting real use of dbus-codegen.

Additionally, added `#[allow(dead_code)]` in two places to silence build warnings.

### CODE REVIEW

File: dbus/src/blocking/generated_org_freedesktop_dbus.rs [hunk 1]
```diff
@@ -116,94 +194,16 @@ impl<'a, T: blocking::BlockingSender, C: ::std::ops::Deref<Target=T>> DBus for b
...
-    fn get_connection_credentials(&self, arg0: &str) -> Result<::std::collections::HashMap<String, arg::Variant<Box<dyn arg::RefArg + 'static>>>, dbus::Error> {
+    fn get_connection_credentials(&self, arg0: &str) -> Result<arg::PropMap, dbus::Error> {
         self.method_call("org.freedesktop.DBus", "GetConnectionCredentials", (arg0, ))
-            .and_then(|r: (::std::collections::HashMap<String, arg::Variant<Box<dyn arg::RefArg + 'static>>>, )| Ok(r.0, ))
+            .and_then(|r: (arg::PropMap, )| Ok(r.0, ))
     }
 
     fn features(&self) -> Result<Vec<String>, dbus::Error> {
-        <Self as blocking::stdintf::org_freedesktop_dbus::Properties>::get(&self, "org.freedesktop.DBus", "Features")
+        <Self as blocking::stdintf::org_freedesktop_dbus::Properties>::get(self, "org.freedesktop.DBus", "Features")
```
Comment (diwic) [inline dbus/src/blocking/generated_org_freedesktop_dbus.rs:203]: Confused about this change, from `&self` to `self`?

File: dbus/src/nonblock/generated_org_freedesktop_dbus.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
-// This code was autogenerated with `dbus-codegen-rust -d org.freedesktop.DBus -p /org/freedesktop/DBus -m none -c nonblock -g -i org.freedesktop. --dbuscrate crate --filter DBus -o dbus/src/nonblock/generated_org_freedesktop_dbus.rs`, see https://github.com/diwic/dbus-rs
-#![allow(missing_docs)]
+// This code was autogenerated with `dbus-codegen-rust -d org.freedesktop.DBus -p /org/freedesktop/DBus -m none -c nonblock -g -i org.freedesktop. --dbuscrate crate --interfaces DBus -o dbus/src/nonblock/generated_org_freedesktop_dbus.rs`, see https://github.com/diwic/dbus-rs
 use crate as dbus;
+#[allow(unused_imports)]
 use crate::arg;
```
Comment (diwic) [inline dbus/src/nonblock/generated_org_freedesktop_dbus.rs:4]: Maybe `#[allow(unused_imports)]` should be a part of what we generate?

File: dbus/src/nonblock/generated_org_freedesktop_dbus.rs [hunk 3]
```diff
@@ -1,6 +1,6 @@
-// This code was autogenerated with `dbus-codegen-rust -d org.freedesktop.DBus -p /org/freedesktop/DBus -m none -c nonblock -g -i org.freedesktop. --dbuscrate crate --filter DBus -o dbus/src/nonblock/generated_org_freedesktop_dbus.rs`, see https://github.com/diwic/dbus-rs
-#![allow(missing_docs)]
```
Comment (diwic) [inline dbus/src/nonblock/generated_org_freedesktop_dbus.rs:2]: Why is `allow(missing_docs)` removed?

**Comment by AldanTanneo** [comment]
All the code in this PR is auto generated by dbus-codegen, as advertised on the first line

**Comment by AldanTanneo** [comment]
But this PR is mostly me being nitpicky, you can close it if it doesn't make sense!

File: dbus/src/nonblock/generated_org_freedesktop_dbus.rs (hunk 2 above)
Comment (AldanTanneo) [reply dbus/src/nonblock/generated_org_freedesktop_dbus.rs:4]: It is! :D

File: dbus/src/blocking/generated_org_freedesktop_dbus.rs (hunk 1 above)
Comment (AldanTanneo) [reply dbus/src/blocking/generated_org_freedesktop_dbus.rs:203]: The self parameter is already a `&self`, so its type is `&Self`, borrowing it again isn't needed

Again, this code is generated by dbus-codegen, so I didn't write that myself

**Pull request merged**